### PR TITLE
Improvements for multiple locales example

### PR DESCRIPTION
**Pull request by hashworks**
Copied from rssbot: https://github.com/iovxw/rssbot/commits/2.x/build.rs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,21 +71,25 @@ Trying to use an unknown key or wrong format arguments is a compile-time error.
...
+    for locale in LOCALES {
+        println!("cargo:rerun-if-changed=locales/{}.toml", locale);
     }
+    println!("cargo:rerun-if-env-changed=LOCALE");
+    let locale_file = format!(
+        "locales/{}.toml",
+        &env::var("LOCALE").unwrap_or("en".to_string())
+    );
+    let out_file = Path::new(&env::var("OUT_DIR").unwrap()).join("strings.rs");
+    let _ignore_error = std::fs::remove_file(&out_file);
```
Comment (GoldsteinE) [inline README.md:93]: I'm not sure that ignoring error is a good thing to do.

**Review by GoldsteinE — CHANGES_REQUESTED** [review]
Please, change documentation in `src/lib.rs` and then regenerate README.md using `cargo-readme`.

File: README.md (hunk 1 above)
Comment (hashworks) [reply README.md:93]: > Please, change documentation in `src/lib.rs` and then regenerate README.md using `cargo-readme`.

Ah, forgot about that. Again. My bad!

> I'm not sure that ignoring error is a good thing to do.

The only errors here are:
* No such file or directory
* Path is a directory
* The user lacks permissions to remove the file

We want to ignore the first error and the other two will cause the macro file creation to fail anyway. Also we have no guarantee that this actually removes the file properly.

Comment (GoldsteinE) [reply README.md:93]: Ok, but probably it should be a comment in code.

**Review by GoldsteinE — APPROVED** [review]

**Pull request merged**